data/remnant/remnant ships.txt

```diff
@@ -230,6 +230,324 @@ ship "Albatross" "Albatross (Ember Shield)"
+	bay "Fighter" -51 -54 under
+	bay "Fighter" 47 -40 under
+	leak "leak" 30 50
+	leak "flame" 30 80
+	leak "big leak" 50 30
+	explode "small explosion" 25
+	explode "medium explosion" 35
+	explode "large explosion" 45
+	explode "huge explosion" 30
+	"final explode" "final explosion large"
+	description "The Tycho Crater Cooperative's Dromedary was one of the foremost colonization ships during the early era before piracy became rampant and triggered the necessity for cityships to be more combat-oriented. These ships incorporate extensive hydroponics, large construction equipment installations, and hundreds of densely packed 'dream cells' built to transport people in bulk."
+	description "	Since the Remnant do not have schematics for the Dromedary, just the highly-modified example of the Shattered Light, this ship is actually a commemorative replica of that specific ship. As such, it combines the original's odd mix of eclectic modifications with new Remnant systems."
+	description "	Unfortunately, since they lack the manufacturing facilities that would normally support a large construction project, it will cost massively more than it would have originally back on Luna."
+
+
+ship "Modified Dromedary" "Modified Dromedary Ghost"
+	sprite "ship/modified dromedary ghost"
+	thumbnail "thumbnail/modified dromedary wreck"
+	"uncapturable"
+	"never disabled"
+	attributes
+		licenses
+			Remnant Capital
+		category "Transport"
+		"cost" 57600000
+		"shields" 5100
+		"hull" 5600
+		"required crew" 95
+		"bunks" 300
+		"mass" 2140
+		"drag" 16.1
+		"heat dissipation" .6
+		"fuel capacity" 900
+		"cargo space" 550
+		"outfit space" 740
+		"weapon capacity" 270
+		"engine capacity" 195
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		"asteroid scan power" 42
+		"atmosphere scan" 100
+		weapon
+			"blast radius" 260
+			"shield damage" 2600
+			"hull damage" 1300
+			"hit force" 3900
+		"automaton" 1
+		inscrutable 1
+		"quantum keystone" 1
+		"burn protection" 100
+		"corrosion protection" 100
+		"discharge protection" 1
+		"disruption protection" 100
+		"force protection" 100
+		"fuel protection" 100
+		"hull protection" 100
+		"ion protection" 1
+		"leak protection" 100
+		"piercing protection" 100
+		"shield protection" 100
+		"slowing protection" 1
+	outfits
+		"Fusion Reactor"
+		"LP144a Battery Pack"
+		"D94-YV Shield Generator"
+		"Large Radar Jammer"
+		"Ramscoop"
+		"NDR-114 Android" 100
+		"Laser Rifle" 100
+		
+		"X4700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Hyperdrive"
+
+	engine 32.5 207
+	engine 3 207
+	engine -27.5 207
+	gun 65 -111 
+		angle 40
+	gun -72 -97.5 
+		angle 320
+	gun -66.5 61 
+		angle 270
+	gun 78 64 
+		angle 90
+	turret 10 -136.5
+	turret -21.5 -57.5
+	turret 51 101
+	turret -63.5 120
+	turret 52.5 143
+		
+
+ship "Modified Dromedary" "Modified Dromedary Specter"
+	sprite "ship/modified dromedary specter"
+	thumbnail "thumbnail/modified dromedary wreck"
+	"uncapturable"
```

Comment:
Technically redundant with never disabled.
```suggestion
```

Reply:
While technically redundant to the extent that these are used in the current missions, the next set of Shattered Light missions will be re-using these, and I plan on having at least a few instances of them being disabled by mission parameters; so I'd prefer to keep these as "uncapturable" to be sure that they are never captured.